numpy/lib/tests/test_histograms.py

```diff
@@ -794,3 +794,21 @@ def test_density_non_uniform_1d(self):
         hist_dd, edges_dd = histogramdd((v,), (bins,), density=True)
         assert_equal(hist, hist_dd)
         assert_equal(edges, edges_dd[0])
+
+    def test_density_via_normed(self):
+        # normed should simply alias to density argument
+        v = np.arange(10)
+        bins = np.array([0, 1, 3, 6, 10])
+        hist, edges = histogram(v, bins, density=True)
+        hist_dd, edges_dd = histogramdd((v,), (bins,), density=None,
```

Comment:
Better to let `density` be the default, rather than passing it explicitly at the same time as normed.